tests/unit/test_detect_target_brightspot.py

```diff
@@ -0,0 +1,201 @@
+"""
+Test DetectTargetBrightspot module.
+"""
+
+import pathlib
+
+import cv2
+import numpy as np
+import pytest
+
+from modules import detections_and_time
+from modules import image_and_time
+from modules.common.modules.logger import logger
+from modules.detect_target import detect_target_brightspot
+
+
+NUMBER_OF_IMAGES = 7
+TEST_PATH = pathlib.Path("tests", "brightspot_example")
+IMAGE_FILES = [pathlib.Path(f"ir{i}.png") for i in range(1, NUMBER_OF_IMAGES + 1)]
+EXPECTED_DETECTIONS_PATHS = [
+    pathlib.Path(TEST_PATH, f"bounding_box_ir{i}.txt") for i in range(1, NUMBER_OF_IMAGES + 1)
+]
+
+DETECTION_TEST_CASES = []
+NO_DETECTION_TEST_CASES = []
+for image_file, expected_detections_file in zip(IMAGE_FILES, EXPECTED_DETECTIONS_PATHS):
+    if expected_detections_file.exists() and expected_detections_file.stat().st_size > 0:
+        DETECTION_TEST_CASES.append((image_file, expected_detections_file))
+    else:
+        NO_DETECTION_TEST_CASES.append(image_file)
```

Comment:
Just rename the images with no detections to something else (e.g. `ir_no_detections_0.png` ).